### DISCUSSION TITLE
how to enable preview features in keycloak 15.0.2 dev server setup

### DISCUSSION THREAD

**Discussion by roypratik0110** [keycloak/keycloak#23810] [Q&A]
i have used this installation and server setup:
https://cloudinfrastructureservices.co.uk/install-keycloak-sso-on-ubuntu-20-04/
server is a aws linux
I want to use TOKEN_EXCHANGE feature, but not been able to access it. please help

**Comment by ThoreKr** [comment]
This is a rock-old version of keycloak using the deprecated wildfly distribution.
You should follow the official documentation to setup the latest quarkus based distribution.
